### PR TITLE
Facebook 1.0.11

### PR DESCRIPTION
**Pull request by JohnNilsson**
Tried to follow the example from the 1.0.10 PR. Rebasing the latest facebook changes on top of master.

Removed the source-map-loader since that wasn't part of the official configuration. Not sure if it is needed or not.

For testing, added [this commit](https://github.com/JohnNilsson/create-react-app-typescript/commit/0dd843346981ccd9e3a2902db7b1cbaae65cc669) (needed some es6 features ts couldn't transpile to es5, so upgraded uglify), and upgraded another project to use this repo. `npm run build` and `npm start` seems to do whats expected of it.

### CODE REVIEW

**Comment by wmonk** [comment]
Thanks this is awesome! Let me have a proper look over.

**Comment by gyzerok** [comment]
Hi @wmonk! Any progress with this update?

**Comment by wmonk** [comment]
Hey sorry, have been super busy. I need to do a proper review (which I had started 😄 ) over the next couple of days!

**Comment by JohnNilsson** [comment]
By the way, I started the merge effort thinking I needed it to get the latest uglifyjs working with webpack. Turns out it didn't I had to resort to disabling it entierly. Not that it should impact the review, just in case someone had similar thoughts.

I think, in the end, I'll probably have to add a babel transpilation step. Would a PR enabling babel (basically restoring the setup from upstream) be accepted?

File: packages/react-scripts/config/webpack.config.dev.js [hunk 1]
```diff
@@ -145,99 +145,84 @@ module.exports = {
         include: paths.appSrc,
       },
       {
-        test: /\.js$/,
```
[the remark this answers is not in the document]
Comment (JohnNilsson) [reply packages/react-scripts/config/webpack.config.dev.js]: I'm currently occupied by other priorities, and can't really promise a deadline. But have some hope to get back to this in a week or two

**Comment by JohnNilsson** [comment]
rebased on master, and readded source-map-loader

**Pull request merged**